### PR TITLE
Test pool migration script

### PR DESCRIPTION
**Pull request by shankar-pankhania-hmcts**

### CODE REVIEW

File: Postgres old schema to new schema migration/test/0015 - pool/015 - pool test setup.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+delete from juror.unique_pool;
+
+insert into juror.unique_pool (owner,pool_no,jurisdiction_code,return_date,next_date,no_requested,pool_total,reg_spc,pool_type,loc_code,new_request,last_update,read_only,deferrals_used,additional_summons,attend_time) values
```
Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/test/0015 - pool/015 - pool test setup.sql:3]: Add another test where only one copy of a unique pool exists and read_only  = 'Y'

**Review by CCHAPMAN01 — CHANGES_REQUESTED** [review]
Please see review comments

File: Postgres old schema to new schema migration/test/0015 - pool/015 - pool test verification.sql [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    attend_time timestamp(0) null,
+    nil_pool bool null,
+    total_no_required int4 not null,
+    date_created timestamp null);
+
+INSERT INTO temp_pool (pool_no,"owner",return_date,no_requested,pool_type,loc_code,new_request,last_update,additional_summons,attend_time,nil_pool,total_no_required,date_created) VALUES
+('4349','400','2024-04-04',1,'CRO','417','Y','2024-04-04 11:33:37.000',1,'2024-04-04 11:33:37.000',false,0,NULL),
+('4345','415','2024-04-04',1,'CRO','417','Y','2024-04-04 11:33:37.000',1,'2024-04-04 11:33:37.000',false,0,NULL),
+('4347','417','2024-06-06',0,'CIV',NULL,NULL,NULL,NULL,'2024-06-04 11:33:37.000',true,0,NULL),
+('4346','416','2024-05-04',NULL,NULL,'440','Y','2024-04-02 11:33:37.000',2,NULL,false,0,NULL);
```
Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/test/0015 - pool/015 - pool test verification.sql:22]: no_requested is now always defaulted to 0 as a new migration stage - please update expected

Comment (KatieHarrison1) [reply Postgres old schema to new schema migration/test/0015 - pool/015 - pool test verification.sql]: Value has now been updated :)

**Pull request merged**